### PR TITLE
[Shape Detection article] Move Barcode Detection up

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Barcode is supported w/o flag, so move it first.

### CODE REVIEW

File: src/site/content/en/blog/shape-detection/index.md [hunk 1]
```diff
@@ -224,6 +201,29 @@ try {
 }
 ```
 
+### Working with the `FaceDetector` {: #facedetector}
+
+The `FaceDetector` always returns the bounding boxes of faces it detects in
+the `ImageBitmapSource`. Depending on the platform, more information
+regarding face landmarks like eyes, nose, or mouth may be available.
```
Comment (jpmedley) [inline src/site/content/en/blog/shape-detection/index.md:208]: Is this API more about finding where the faces are than about identifying people? Maybe we should say something about that.

Comment (tomayac) [reply src/site/content/en/blog/shape-detection/index.md:208]: There's an important difference in literature between face detection (there's *a* face) and face identification (there's a face, and it's *Joe's*). This API is only about detection.

File: src/site/content/en/blog/shape-detection/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ authors:
   - thomassteiner
 description: The Shape Detection API detects faces, barcodes, and text in images.
 date: 2019-01-07
-updated: 2020-06-04
+updated: 2020-07-31
```
Comment (tomayac) [inline src/site/content/en/blog/shape-detection/index.md:8]:
```suggestion
updated: 2020-08-10
```

File: src/site/content/en/blog/shape-detection/index.md (hunk 1 above)
Comment (tomayac) [inline src/site/content/en/blog/shape-detection/index.md:208]:
```suggestion
regarding face landmarks like eyes, nose, or mouth may be available.
It is important to note that this API only detects faces.
It does not identify who a face belongs to.
```

**Pull request merged**